openshift-start-configured-cluster

```diff
@@ -43,6 +43,17 @@ for i in openshift-apiserver openshift-controller-manager; do
     sed -i 's|webhookTokenAuthenticators: null|webhookTokenAuthenticators:\n  - configFile: "webhook.yaml"|' $OC_CONFIG/$i/master-config.yaml
 done
 
+echo "Generating Keycloak client cert"
+# This need to happen before the oc cluster
+openssl genrsa -out $OC_CONFIG/openshift-apiserver/keycloak-client.key 2048
+openssl req -new -key $OC_CONFIG/openshift-apiserver/keycloak-client.key -out $OC_CONFIG/openshift-apiserver/keycloak-client.csr -subj "/CN=api.keycloak.localhost"
```

Comment:
Please use a different CN if create-master-certs is not ok.
CN=api.keycloak.localhost is not a valid name for the API *server*.
You should use something like: CN=kubernetes.default.svc.luster.local or maybe  CN=api.k8s.io if you want somethign short.
The client certis identifying the Kube API server *to* keycloak, not the other way around, so the right name makes it clear this client is the kube api server.

Reply:
Done in https://github.com/keycloak/openshift-integration/pull/18